instances/subjectAttribute/knockin.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
all "null"s are marked in red for some reason... you can double check but I think nothing is wrong...

Reply:
see above